src/avif.c

```diff
@@ -1170,6 +1170,7 @@ void avifGainMapDestroy(avifGainMap * gainMap)
     if (gainMap->image) {
         avifImageDestroy(gainMap->image);
     }
+    avifRWDataFree(&gainMap->altICC);
```

Comment:
uh, actually this should be moved to  avifGainMapDestroy() and the call below should be avifGainMapDestroy() as well

Reply:
sorry I misread the github diff this is actually what you're doing -_- nevermind!